perception/detected_object_validation/src/obstacle_pointcloud_based_validator.cpp

```diff
@@ -37,6 +37,17 @@
 
 namespace
 {
+template <typename T>
+inline std::map<uint8_t, T> convertListToMap(const std::vector<T> & input_vector)
+{
+  std::map<uint8_t, T> output_map;
+  uint8_t label = 0;
+  for (const T & v : input_vector) {
+    output_map.insert(std::make_pair(label, v));
+    label++;
+  }
+  return output_map;
+}
```

Comment:
I think `std::vector`  just meets what you want to do.

Reply:
@yukke42 thank you for your comment. I fixed at https://github.com/autowarefoundation/autoware.universe/pull/5177/commits/b94e6e638deb079c3c952276cd961aadee1639a7

Reply:
@yukke42 Do you have further comment? Otherwise, I will merge this PR.